### PR TITLE
beta.1 upgrade

### PR DESCRIPTION
**Pull request by vmidyllic**
Changes from beta.0 to beta.1 version of circuit. New operators.
Do not merge until it will be in mobile app.

### CODE REVIEW

**Review by volodymyr-basiuk — APPROVED** [review]

**Comment by amonsosanz** [comment]
Do we need to update the circuits on the query builder?

**Comment by amonsosanz** [comment]
I guess we should update this page too right?

https://devs.polygonid.com/docs/verifier/query-builder

File: docs/verifier/v3-circuit.md [hunk 1]
```diff
@@ -18,10 +18,27 @@ We want to introduce new circuits - СredentialAtomicQueryV3 and CredentialAtomi
...
     Circuits are in the beta version. Trusted setup will be performed in the next release.
-    Current Circuit id is `СredentialAtomicQueryV3-beta.0`
+    Newest version is СredentialAtomicQueryV3-beta.1
+
+  <details><summary>Changelog from beta.0 to beta.1</summary>
+ 
+
+    1. ValueArrSize input is introduced, which fixes behavoiur for IN / NIN operations. 
+    2. Exists operator support.
+    3. Noop operator support.
```
Comment (amonsosanz) [inline docs/verifier/v3-circuit.md:28]: I think we should describe the utility of this attribute. Something in the line:

Noop enables a verifier to request proof of the presence of a credential in the user's wallet without requiring any additional conditions to be met or revealing any specific details of the credential.

File: docs/verifier/verification-library/zk-query-language.md [hunk 2]
```diff
@@ -495,6 +894,32 @@ Please, check out our [<ins>Onchain verification tutorial to build a query</ins>
...
 }
 ```
+</TabItem>
+</Tabs>
+
+
+
+## Proof of credential issuance
+
+Feature that enables requests to prove that credential exits in the ID Holder wallet without revealing any info.
```
Comment (amonsosanz) [inline docs/verifier/verification-library/zk-query-language.md:904]: Shouldn't we mention here that this query somehow relates to the `$noop` operator?

Also, I'd consider a rephrasing of this description to:

By not including the `credentialSubject` in the query, a verifier requests proof of the presence of a credential in the user's wallet without requiring any condition to be met or revealing any specific detail about the credential.

**Comment by vmidyllic** [comment]
> I guess we should update this page too right?
> 
> https://devs.polygonid.com/docs/verifier/query-builder

No, see nothing to update there

File: docs/verifier/verification-library/zk-query-language.md (hunk 2 above)
Comment (vmidyllic) [reply docs/verifier/verification-library/zk-query-language.md:904]: fixed to subject

File: docs/verifier/verification-library/zk-query-language.md [hunk 3]
```diff
@@ -495,6 +894,32 @@ Please, check out our [<ins>Onchain verification tutorial to build a query</ins>
...
+
+**Query**
+
+In the example below, the verifier requests the Holder's country code.
```
[the remark this answers is not in the document]
Comment (vmidyllic) [reply docs/verifier/verification-library/zk-query-language.md:908]: fixed

File: docs/verifier/v3-circuit.md (hunk 1 above)
Comment (vmidyllic) [reply docs/verifier/v3-circuit.md:28]: added link bellow the summary box.

File: docs/verifier/verification-library/zk-query-language.md [hunk 4]
```diff
@@ -29,6 +29,14 @@ The Query Language sits on top of these circuits to provide a simple way for dev
 - Must not be a resident of a country in the list of blacklisted countries to operate on an exchange - `not-in` (operator 5).
 - Must not be a resident of a specific country - `not-equal` (operator 6).
 
+The <a href="https://docs.iden3.io/protocol/main-circuits/#credentialatomicquerysigv2onchain:~:text=verify%20the%20proof.-,credentialAtomicQueryV3,-Github" target="_blank">Atomic Query Signature V3 Circuit</a> circuit is designed to support more operators:
```
[the remark this answers is not in the document]
Comment (vmidyllic) [reply docs/verifier/verification-library/zk-query-language.md:32]: fixed

**Comment by amonsosanz** [comment]
> > I guess we should update this page too right?
> > https://devs.polygonid.com/docs/verifier/query-builder
> 
> No, see nothing to update there

Shouldn't the new operators be added [here](https://devs.polygonid.com/docs/verifier/query-builder/#operator)?

**Review by amonsosanz — APPROVED** [review]

**Pull request merged**